test/integration/controller/admissionchecks/provisioning/suite_test.go

```diff
@@ -53,33 +57,39 @@ func TestProvisioning(t *testing.T) {
 	)
 }
 
-func managerSetup(mgr manager.Manager, ctx context.Context) {
-	err := indexer.Setup(ctx, mgr.GetFieldIndexer())
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+func managerSetup(opts ...provisioning.Option) framework.ManagerSetup {
+	return func(mgr manager.Manager, ctx context.Context) {
```

Comment:
Maybe just this?
```suggestion
func managerSetup(mgr manager.Manager, ctx context.Context, opts ...provisioning.Option) {
```

Reply:
Unfortunately no. The `ManagerSetup` that using on `fwk.RunManager` required only this two fields.

```go
type ManagerSetup func(manager.Manager, context.Context)
```